x509/src/Cardano/X509/Configuration.hs

```diff
@@ -304,27 +330,22 @@ usExtensionsV3 purpose subDN issDN =
         ]
 
 
-svExtensionsV3 :: DistinguishedName -> DistinguishedName -> NonEmpty String -> [ExtensionRaw]
-svExtensionsV3 subDN issDN altNames =
-    let
-        subjectAltName = ExtSubjectAltName ( parseAltName <$> NonEmpty.toList altNames)
-    in
-        extensionEncode False subjectAltName : usExtensionsV3 KeyUsagePurpose_ServerAuth subDN issDN
+svExtensionsV3 :: MonadThrow m => DistinguishedName -> DistinguishedName -> NonEmpty String -> m [ExtensionRaw]
+svExtensionsV3 subDN issDN altNames = do
+    subjectAltName <-
+        ExtSubjectAltName <$> mapM parseAltName (NonEmpty.toList altNames)
+
+    return $
+        extensionEncode False subjectAltName :
+        usExtensionsV3 KeyUsagePurpose_ServerAuth subDN issDN
+
+
+parseAltName :: MonadThrow m => String -> m AltName
+parseAltName name =
+    case IP.decode (T.pack name) of
+        Just _  -> throwM (ErrInvalidAltName "IP was given as an AltName but it should be a DNS name")
```

Comment:
This will break deadalus. Please leave the IP SAN parsing in.

Reply:
Hmmm... Good catch @disasm 
But IP SAN aren't _recognized as a valid alternative names_ by the x509 library we use. I can perhaps tweak the hook which validates the name...

Does Daedalus uses 127.0.0.1 to contact the server instead of `localhost`?